web/src/main/java/com/epam/esm/config/SecurityConfig.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.esm.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpMethod;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.annotation.web.configurers.oauth2.server.resource.OAuth2ResourceServerConfigurer;
+
+//@Configuration
```

Comment:
You should not store commented code under vcs on no circumstances. Better to store separately or at least made commit + revert commit in case you want to store it in vcs history